packages/framework/src/Model/Product/Accessory/ProductAccessoryFacade.php

```diff
@@ -31,4 +33,19 @@ public function getTopOfferedAccessories(Product $product, $domainId, PricingGro
     {
         return $this->productAccessoryRepository->getTopOfferedAccessories($product, $domainId, $pricingGroup, $limit);
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
+     * @param int $domainId
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroup $pricingGroup
+     * @return array
+     */
+    public function getAllAccessories(Product $product, int $domainId, PricingGroup $pricingGroup): array
```

Comment:
This method exist because of FEAPI only - should not it be in its package?

Reply:
Nice catch, just sorry, I fixed it in commit not in fixup, I hope it is not a problem